### PR TITLE
update basic video

### PR DESCRIPTION
**Pull request by abdulajet**
I moved the min ios version to ios 12, cleaned up all the project warnings and deprecations that arose too.

### CODE REVIEW

File: Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -311,7 +312,6 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
```
Comment (jvalli) [inline Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj:314]: Why did you remove `LaunchImage`?

File: Basic-Video-Chat/Basic-Video-Chat/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -2,92 +2,97 @@
   "images" : [
     {
       "idiom" : "iphone",
-      "size" : "20x20",
```
Comment (jvalli) [inline Basic-Video-Chat/Basic-Video-Chat/Images.xcassets/AppIcon.appiconset/Contents.json:5]: There is no real change on this file, just switching order of key/values. Is there any reason to push this?

Comment (abdulajet) [reply Basic-Video-Chat/Basic-Video-Chat/Images.xcassets/AppIcon.appiconset/Contents.json:5]: Yeah I realised after pushing, this is something Xcode did. Not sure if they have changed something there

File: Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj (hunk 1 above)
Comment (abdulajet) [reply Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj:314]: I believe that `LaunchImage` has been deprecated. It was throwing an error/warning. The file in the assets folder was also empty so I got rid of it.

Comment (jvalli) [reply Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj:314]: 👍

File: Basic-Video-Chat/Basic-Video-Chat/Images.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (jvalli) [reply Basic-Video-Chat/Basic-Video-Chat/Images.xcassets/AppIcon.appiconset/Contents.json:5]: If there is no change, can we removed this from the PR please?

**Review by jvalli — APPROVED** [review]
